dspace/config/modules/bitstream.cfg

```diff
@@ -0,0 +1,5 @@
+#------------------------------------------------------------#
+#-----------------Access type configuration -----------------#
+#------------------------------------------------------------#
+bitstream.upload.default.accesstype.enabled = true
```

Comment:
`false` into `uclouvain-base`, create a PR into uclouvain-pr branch to activate it